backend/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "cross-env NODE_ENV=test jest --runInBand",
     "start": "cross-env NODE_ENV=dev nodemon ./src/server.ts",
+    "start-migrate": "cross-env NODE_ENV=dev MIGRATE=true nodemon ./src/server.ts",
```

Comment:
is this meant to be run once (and the command will later be removed) or every time?

Reply:
in the first case, migration would be an interesting topic for the next team meeting. it definitely wasn't covered in COMP303 but comes up in real-world applications

Reply:
so its just an alternate command you can use to run the application for dev purposes. "npm start" will clear the database when it connects, "npm run start-migrate" will create some sample data in the database when it starts up, just for dev / testing purposes, so you dont have to manually create some artists, boroughs, murals, etc every time you want to try the api. If you close the app then run "npm start", that data will be gone and you will have an empty database. If you think it is better, we can make it such that "npm start" does not clear your personal database, and as such your data persists between runs of the backend, but then if you want to clear the database you have to connect to the container yourself.

Reply:
Now we will have to work on migrating the existing data from airtable they gave us to a production database at some point, but that has not been worked on yet. how soon do you think we should work on setting up deployment / production environment? asap?

Reply:
Late response! The alternative start script with mock data for dev makes sense.

As for migration, Lea gave us access to the Airtable. Soph suggested a one-time Python script or similar to do the migration. We're in no rush for production data though - the next priority will be having basic full-stack functionality.